js/src/lib/components/InferenceWidget/widgets/AudioToAudioWidget/AudioToAudioWidget.svelte

```diff
@@ -62,7 +62,11 @@
 		}
 	}
 
-	async function getOutput({ withModelLoading = false, isOnLoadCall = false }: InferenceRunFlags = {}) {
+	async function getOutput({
+		withModelLoading = false,
+		isOnLoadCall = false,
+		exampleOutput = undefined,
```

Comment:
This has no effect in this widget

Reply:
yes, because widgets such as AudioToAudio or ImageSegmentation are not expected to have `example.output` in their model cards

for example here https://github.com/huggingface/hub-docs/blob/e5cdf0af4ea36411f489c290e6af144e43c7723c/js/src/lib/components/InferenceWidget/widgets/AudioToAudioWidget/AudioToAudioWidget.svelte#L131, if they were expected to have `example.output`, the typing would have been `sample: WidgetExampleAssetInput<WIDGET OUTPUT TYPE>`

docs here for the list of supported output types: https://huggingface.co/docs/hub/models-widgets#example-outputs from #978

@julien-c could you confirm this statement
> yes, because widgets such as AudioToAudio or ImageSegmentation are not expected to have `example.output` in their model cards

Reply:
i'm not sure i understand, i'm a bit lost...

Reply:
from the docs (emphasis mine):
> **each** widget example can also optionally describe the corresponding model output, directly in the output property

AudioToAudio or ImageSegmentation should support a file output, i.e. a `output.url`, same as text-to-image:

<img width="1028" alt="image" src="https://github.com/huggingface/hub-docs/assets/326577/a6366a14-fa12-42a9-ae8c-3c3c02d53037">

Reply:
As we discussed internally, lets merge this PR without the proposed changes above (`remaining widgets validate and use output`). And iterate in subseq PRs

Reply:
> Should I open a PR (extend the current PR) to make all the remaining widgets validate and use `output` ?

Yes

Reply:
that was the goal to support `output` for all widgets